github/repos_collaborators_test.go

```diff
@@ -24,7 +24,35 @@ func TestRepositoriesService_ListCollaborators(t *testing.T) {
 		fmt.Fprintf(w, `[{"id":1}, {"id":2}]`)
 	})
 
-	opt := &ListOptions{Page: 2}
+	opt := &ListCollaboratorsOptions{
+		ListOptions: ListOptions{Page: 2},
+		Affiliation: "",
+	}
+	users, _, err := client.Repositories.ListCollaborators(context.Background(), "o", "r", opt)
+	if err != nil {
+		t.Errorf("Repositories.ListCollaborators returned error: %v", err)
+	}
+
+	want := []*User{{ID: Int(1)}, {ID: Int(2)}}
+	if !reflect.DeepEqual(users, want) {
+		t.Errorf("Repositori es.ListCollaborators returned %+v, want %+v", users, want)
+	}
+}
+
+func TestRepositoriesService_ListCollaborators_With_Affiliation(t *testing.T) {
```

Comment:
Minor. The next test is called `TestRepositoriesService_ListCollaborators_invalidOwner`. Let's make this `TestRepositoriesService_ListCollaborators_withAffiliation` for consistency.

This pattern comes from pattern for naming examples, as described at the bottom of https://godoc.org/testing#hdr-Examples section. We're using it for tests too here.